staging/src/kubevirt.io/api/instancetype/v1alpha1/types.go

```diff
@@ -71,19 +71,6 @@ type VirtualMachineClusterInstancetypeList struct {
 	Items           []VirtualMachineClusterInstancetype `json:"items"`
 }
 
-// VirtualMachineInstancetypeSpecRevision is used to capture a versioned copy of VirtualMachineInstancetypeSpec within a ControllerRevision
-//
-// **WARNING** This struct should never change ensuring it can always be unmarshaled from a ControllerRevision
-type VirtualMachineInstancetypeSpecRevision struct {
```

Comment:
@davidvossel Moving the discussion here, this will break existing users but as we are still in alpha breaking changes like this are still accepted AFAIK [1]. We could signal this with a bump to `v1alpha2` if it would help?

[1] https://kubernetes.io/docs/reference/using-api/#api-versioning

Reply:
FWIW I've also created https://github.com/kubevirt/kubevirt/issues/8235 recently to track getting this API to beta.

Reply:
Increasing the version to `v1alpha2` would not solve the problem that existing ControllerRevisions would not work.
But because this is still alpha feature, could we document somewhere that after this change, old ControllerRevisions need to be removed? Then they will be recreated in the new format.

Reply:
The bump is just to signal that something had changed but yeah I agree that a release note would be useful here.

Reply:
yeah, if we're goign to do this, we need to bump the api version v1alpha2 and have a release note. That should be fine.